tests/defender/system/aws_iot_tests_defender_system.c

```diff
@@ -28,8 +28,13 @@
 #include <stdio.h>
 
 /* Network includes. */
-#include <netdb.h>
-#include <arpa/inet.h>
+#if defined( __GNUC__ )
```

Comment:
See comment in iot_network_mbedtls.c

Also: This file should be cross-platform, so must not use any platform-specific includes.

Reply:
OK will change to #ifndef( _WIN32 ).